travis/run_pylint.py

```diff
@@ -67,7 +41,9 @@ def run_pylint(paths, cfg, beta_msgs=None, sys_paths=None, extra_params=None):
     sys.path.extend(sys_paths)
     cmd = ['--rcfile=' + cfg]
     cmd.extend(extra_params)
-    subpaths = get_subpaths(paths)
+
+    subpaths = getaddons.get_subpaths(paths)
```

Comment:
This isn't conditional on the environment variable. Travis is likely failing due to this https://travis-ci.org/OCA/maintainer-quality-tools/jobs/174463515#L327